src/Spago/GlobalCache.hs

```diff
@@ -65,7 +65,7 @@ globallyCache (packageName, Repo url, ref) downloadDir metadata cacheableCallbac
           cacheableCallback $ Turtle.encodeString resultDir
       where
     _ -> do
-      echo $ "Warning: was not able to match url with GitHub ones: " <> url
+      echo $ "WARNING: Not caching repo, because URL doesn't have the form of 'https://github.com/<ORG>/<REPO>.git': " <> url
```

Comment:
I was also thinking about making the parsing logic more robust (i.e. not to always expect `.git` at the end of url) but I rejected it because I think it's best to have unified format for all packages in the package-set.

Reply:
Yeah the unified format is good, though accepting URLs not containing `.git` should be pretty harmless (and I think the Curator implements this already)

However we don't have to change this now and we can do it in a followup PR